pig/src/main/java/org/jboss/pnc/bacon/pig/impl/utils/FileDownloadUtils.java

```diff
@@ -61,7 +61,6 @@
     public static void downloadTo(URI downloadUrl, File targetPath) {
         log.info("Downloading {} to {}", downloadUrl, targetPath);
```

Comment:
maybe put this as debug? user is still able to debug with --verbose

Reply:
true true. will change it to debug